dashboard/src/actions/charts.ts

```diff
@@ -93,10 +96,16 @@ export function fetchCharts(
   query?: string,
 ): ThunkAction<Promise<void>, IStoreState, null, ChartsAction> {
   return async dispatch => {
-    dispatch(requestCharts(query));
+    dispatch(requestCharts(page));
     try {
-      const charts = await Chart.fetchCharts(cluster, namespace, repos, page, size, query);
-      dispatch(receiveCharts(charts));
+      const response = await Chart.fetchCharts(cluster, namespace, repos, page, size, query);
+      dispatch(
+        receiveCharts({
+          items: response.data,
+          page,
+          totalPages: response.meta.totalPages,
+        } as IReceiveChartsActionPayload),
```

Comment:
Yes, this is not necessary at all, but I just wanted to type the object so that we can get transpiler errors when changing its props in the future. However, I'm ok removing it.